backend/core/src/listings/combined/combined-listing-transformer.ts

```diff
@@ -0,0 +1,96 @@
+import { Jurisdiction } from "../../jurisdictions/entities/jurisdiction.entity"
+import { ListingMultiselectQuestion } from "../..//multiselect-question/entities/listing-multiselect-question.entity"
+import { ReservedCommunityType } from "../../reserved-community-type/entities/reserved-community-type.entity"
+import { Address } from "../../shared/entities/address.entity"
+import { mapTo } from "../../shared/mapTo"
+import { Unit } from "../../units/entities/unit.entity"
+import { ListingFeatures } from "../entities/listing-features.entity"
+import { ListingImage } from "../entities/listing-image.entity"
+import { ListingUtilities } from "../entities/listing-utilities.entity"
+import { Listing } from "../entities/listing.entity"
+import { summarizeUnitsByTypeAndRent } from "../../shared/units-transformations"
+
+/**
+ * Converts raw table row from combined_listings view into a Listing object
+ * REMOVE_WHEN_EXTERNAL_NOT_NEEDED
+ */
+export class CombinedListingTransformer {
```

Comment:
I intentionally kept them as separate as possible to 1) reduce the chance of side effects, 2) make it as easy as possible to remove all of this in a few months when it is no longer needed, and 3) because the useful overlap between the two is actually really small. 

More importantly, `ExternalListing` is not the same as `Listing`; they represent different things, are stored in different tables, and have different lifecycles.  TypeORM should honestly not even know about `ExternalListing`, because it is never queried directly anywhere in the code (all SELECTs are against `CombinedListing`) and the ownership of all data in that table exists outside of `backend/core`.  Any savings in LOC would not be worth the additional maintenance burden, IMO